gcn/notices/fermi/gbm/detectors.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/detectors.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "",
+  "description": "",
+  "type": "object",
+  "properties": {
+    "detectors_status": {
+      "type": "array",
+      "items": {
+        "$ref": "../../core/DetectorStatus.schema.json"
```

Comment:
These can be referenced by name rather than indexed, which would then not require this detectors schema at all.  See https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/burstcube/alert.example.json or https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/glowbug/Alert.schema.json, which you previously came up with.

Reply:
I think we are discussing two design points here:

1. Array vs. names;
2. To `$ref` or not to `$ref` the `core/DetectorStatus`.

Thanks for suggesting an alternative to the array. The two choices were discussed at our pipeline meeting (1 vote for names) and the IOTL meeting. The 14-array won. Consider that it is a bit mask in the data products, in which the bits are indexed 0-13.  The same is in the classic notes, incl. the text ones. Also the old and the new GBM tools use straight arrays. In retrospect, I should had used an array for Glowbug as well.

On point (2), if you would like to remove the `$ref`, I'll be happy to replace it with the `enum`. just let me know.

Reply:
Hi @shb46, Could you please implement detectors schema within `alert.schema.json`?
I understand the logic of making it as a library and calling it. But it's not a few lines field and would make `alert.schema.json` look a self consistent schema.

Are you planning to add more topics in future and wanna call `detectors.schema.json`?

Reply:
@Vidushi-GitHub: The detectors schema, if we keep it, will be shared with another topic. Do you still want me to make it inline?

Reply:
@shb46 Ok, thanks. If you plan to have more topics related to Fermi/GBM, then that's fine.
Please remember to limit topics per mission to simplify user subscriptions to new Notices. The new Schema allows multiple Notices using a single Schema since fields are not mandatory.

Reply:
@Vidushi-GitHub: Our intent was for GBM to have 5 topics.  Hope that is not too much.  I suspect we might need a 6th one.  Separately, there might be topics for internal distribution only but this hasn't been discussed yet.

My plan is to have a 1-to-1-to-1 correspondence of classic notice types, schemas and topics. I think it is a little too late to steer towards making a single schema. We are ahead with approving the ALERT schema and it will set us back. My deadline is Dec 12.

Reply:
@shb46 The design of the unified schema has always been to incorporate many of the classic notice types into the same topic so users can only follow 1 or 2 topics per instrument.  You can send updates as more information to fill out as the schema becomes available, but rebuilding GCN Classic in the JSON-format was never the intention. We should discuss the deadline offline, as I assume that's tied to senior review.

Reply:
@jracusin: No problem, I can use topic `gcn.notices.fermi.gbm` for everything.